src/components/shared/formItems/ProjectBondingCurveRate.tsx

```diff
@@ -42,7 +42,8 @@ function BondingCurveRateExtra({ disabled }: { disabled?: boolean }) {
       {disabled && (
         <FormItemWarningText>
           <Trans>
-            Disabled when your project's funding cycle has no funding target.
+            Disabled when your project's distribution limit is <i>No limit</i>{' '}
```

Comment:
**suggestion** can we bold this? We don't have a style guide but I think bold is generally the recommended approach in cases like this.

Reply:
I tried it. It looks pre nasty IMO. Italic was better I thought

Reply:
Actually nah, on second look, bold looks better